src/components/Navbar.js

```diff
@@ -75,7 +76,8 @@ export class Navbar extends Component {
                   </DropdownMenu>
                 </UncontrolledDropdown>
               </Nav>
-            </Collapse>)}
+            </Collapse>
```

Comment:
Is this a formatting change?

Reply:
@Duncanian, there has been an issue everyone is faceing from travis that makes us make this change.